src/connection/Connection.ts

```diff
@@ -419,18 +419,18 @@ export class Connection {
 
         return relationMetadata.junctionEntityMetadata;
     }
-
-    // -------------------------------------------------------------------------
-    // Protected Methods
-    // -------------------------------------------------------------------------
-
+    
     /**
      * Creates an Entity Manager for the current connection with the help of the EntityManagerFactory.
      */
-    protected createEntityManager(queryRunner?: QueryRunner): EntityManager {
+    createEntityManager(queryRunner?: QueryRunner): EntityManager {
```

Comment:
what is the purpose of this method being public? I don't really want to confuse users with both `getEntityManager` and `createEntityManager`. Plus there is a createQueryRunner which creates an entity as well...

Reply:
I added this method to resolve a circlular dependency. And the [EntityManager](https://github.com/daniel-lang/typeorm/blob/sqljs/src/entity-manager/EntityManager.ts#L95) also uses the method...